### PR TITLE
Changes Google

### PR DESCRIPTION
**Pull request by gallegonovato**

### CODE REVIEW

File: pihole-google.txt [hunk 1]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+g.doubleclick.net
+imasdk.googleapis.com
+pagead2.googlesyndication.com
+partner.googleadservices.com
+googleoptimize.com
+googletagmanager.com
+google-analytics.com
+googleadservices.com
+googlerank.info
+www-google-analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7553]:
```suggestion

```

File: pihole-google.txt [hunk 2]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+doubleclick-net.com
+doubleclickbygoogle.com
+googleapi.club
+googletagservices.com
+imasdk.googleapis.com
+googletrackmanager.compartner.googleadservices.com
+partner.video.syndication.msn.com
+googleadapis.l.google.com
+googleads.github.io
+gstaticadssl.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7540]:
```suggestion
```

File: pihole-google.txt [hunk 3]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+pagead-googlehosted.l.google.com
+firebase.com
+firestore.googleapis.com
+firebaseremoteconfig.googleapis.com
+firebaseinstallations.googleapis.com
+firebaselogging.googleapis.com
+firebase-settings.crashlytics.com
+try.crashlytics.com
+settings.crashlytics.com
+crashlytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7529]:
```suggestion
```

File: pihole-google.txt [hunk 4]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+pubads.g.doubleclick.net
+www.googletagservices.com
+adservice.google.com
+adservice.io
+googleadservices.com
+feedads.googleadservices.com
+partner.googleadservices.com
+www.googleadservices.com
+9098b63249ee4522c5d8677059fe22e4.safeframe.googlesyndication.com
+pagead-googlehosted.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7520]:
```suggestion
```

File: pihole-google.txt [hunk 5]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+crashlytics.l.google.com
+crashlytics.com
+doubleclick-net.com
+doubleclickbygoogle.com
+googleapi.club
+googletagservices.com
+imasdk.googleapis.com
+googletrackmanager.compartner.googleadservices.com
+partner.video.syndication.msn.com
+googleadapis.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7538]:
```suggestion
```

File: pihole-google.txt [hunk 6]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+stats.l.doubleclick.net
+m.doubleclick.net
+s0.doubleclick.net
+static.doubleclick.net
+google-analytics.com
+ssl.google-analytics.com
+www.google-analytics.com
+ads.google.com
+adservice.google.com
+crashlytics2.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7603]:
```suggestion
```

File: pihole-google.txt [hunk 7]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+s0-2mdn-net.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7604]:
```suggestion
```

File: pihole-google.txt [hunk 8]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+pagead2.googlesyndication.com
+tpc.googlesyndication.com
+displayads-formats.googleusercontent.com
+tpc.googlesyndication.com
+google-analytics.com
+www.google-analytics.com
+analytics.withgoogle.com
+googleanalytics.com
+ssl.google-analytics.com
+www-google-analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7621]:
```suggestion
```

File: pihole-google.txt [hunk 9]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+ssl-google-analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7622]:
```suggestion
```

File: pihole-google.txt [hunk 10]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+google-analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7623]:
```suggestion
```

File: pihole-google.txt [hunk 11]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+mimageads5.googleadservices.com
+mimageads6.googleadservices.com
+mimageads7.googleadservices.com
+mimageads8.googleadservices.com
+mimageads9.googleadservices.com
+adsense.google.com
+www.googleoptimize.com
+googleadapis.l.google.com
+googleads.github.io
+gstaticadssl.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7682]:
```suggestion
```

File: pihole-google.txt [hunk 12]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+mimageads3.googleadservices.com
+mimageads4.googleadservices.com
+mimageads5.googleadservices.com
+mimageads6.googleadservices.com
+mimageads7.googleadservices.com
+mimageads8.googleadservices.com
+mimageads9.googleadservices.com
+adsense.google.com
+www.googleoptimize.com
+googleadapis.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7680]:
```suggestion
```

File: pihole-google.txt [hunk 13]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+pagead-googlehosted.l.google.com
+firebase.com
+firestore.googleapis.com
+firebaseremoteconfig.googleapis.com
+firebaseinstallations.googleapis.com
+firebaselogging.googleapis.com
+firebase-settings.crashlytics.com
+try.crashlytics.com
+settings.crashlytics.com
+crashlytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7408]:
```suggestion
```

File: pihole-google.txt [hunk 14]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+static.doubleclick.net
+mng-ads.com
+adservice.google.com
+adservice.io
+googleadservices.com
+feedads.googleadservices.com
+partner.googleadservices.com
+www.googleadservices.com
+9098b63249ee4522c5d8677059fe22e4.safeframe.googlesyndication.com
+pagead-googlehosted.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7399]:
```suggestion
```

File: pihole-google.txt [hunk 15]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+settings.crashlytics.com
+crashlytics.l.google.com
+crashlytics.com
+tpc.googlesyndication.com
+google-analytics.com
+www.google-analytics.com
+analytics.withgoogle.com
+googleanalytics.com
+ssl.google-analytics.com
+www-google-analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7416]:
```suggestion
```

File: pihole-google.txt [hunk 16]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+ssl-google-analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7417]:
```suggestion
```

File: pihole-google.txt [hunk 17]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+google-analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7418]:
```suggestion
```

File: pihole-google.txt [hunk 18]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+mimageads7.googleadservices.com
+mimageads8.googleadservices.com
+mimageads9.googleadservices.com
+adsense.google.com
+www.googleoptimize.com
+choices.consentframework.com
+s.skimresources.com
+pixel.wp.com
+docs.google.com
+analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7461]:
```suggestion
```

File: pihole-google.txt [hunk 19]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+mimageads9.googleadservices.com
+adsense.google.com
+www.googleoptimize.com
+tpc.googlesyndication.com
+google-analytics.com
+www.google-analytics.com
+analytics.withgoogle.com
+googleanalytics.com
+ssl.google-analytics.com
+www-google-analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7489]:
```suggestion
```

File: pihole-google.txt [hunk 20]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+ssl-google-analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7490]:
```suggestion
```

File: pihole-google.txt [hunk 21]
```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
...
+google-analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7491]:
```suggestion
```

**Review by nickspaargaren — CHANGES_REQUESTED** [review]

File: pihole-google.txt (hunk 1 above)
Comment (gallegonovato) [reply pihole-google.txt:7553]: Done

File: pihole-google.txt (hunk 2 above)
Comment (gallegonovato) [reply pihole-google.txt:7540]: Done

File: pihole-google.txt (hunk 3 above)
Comment (gallegonovato) [reply pihole-google.txt:7529]: Done

File: pihole-google.txt (hunk 4 above)
Comment (gallegonovato) [reply pihole-google.txt:7520]: Done

File: pihole-google.txt (hunk 5 above)
Comment (gallegonovato) [reply pihole-google.txt:7538]: Done

File: pihole-google.txt (hunk 7 above)
Comment (gallegonovato) [reply pihole-google.txt:7604]: ??

File: pihole-google.txt (hunk 8 above)
Comment (gallegonovato) [reply pihole-google.txt:7621]: Done

File: pihole-google.txt (hunk 9 above)
Comment (gallegonovato) [reply pihole-google.txt:7622]: Done

File: pihole-google.txt (hunk 10 above)
Comment (gallegonovato) [reply pihole-google.txt:7623]: Done

File: pihole-google.txt (hunk 11 above)
Comment (gallegonovato) [reply pihole-google.txt:7682]: ??

File: pihole-google.txt (hunk 13 above)
Comment (gallegonovato) [reply pihole-google.txt:7408]: Delete, but i will put

File: pihole-google.txt (hunk 1 above)
Comment (nickspaargaren) [reply pihole-google.txt:7553]: Are you sure that you commit your changes?

File: pihole-google.txt (hunk 14 above)
Comment (gallegonovato) [reply pihole-google.txt:7399]: Delete but i left it

File: pihole-google.txt (hunk 15 above)
Comment (gallegonovato) [reply pihole-google.txt:7416]: Delete but i left www-google-analytics.l.google.com

File: pihole-google.txt (hunk 16 above)
Comment (gallegonovato) [reply pihole-google.txt:7417]: Deleted but i left it

File: pihole-google.txt (hunk 17 above)
Comment (gallegonovato) [reply pihole-google.txt:7418]: Deleted but i left it

File: pihole-google.txt (hunk 18 above)
Comment (gallegonovato) [reply pihole-google.txt:7461]: Idem

File: pihole-google.txt (hunk 19 above)
Comment (gallegonovato) [reply pihole-google.txt:7489]: Delete

File: pihole-google.txt (hunk 20 above)
Comment (gallegonovato) [reply pihole-google.txt:7490]: Delete

File: pihole-google.txt (hunk 1 above)
Comment (gallegonovato) [reply pihole-google.txt]: Now??

File: pihole-google.txt (hunk 6 above)
Comment (gallegonovato) [reply pihole-google.txt:7589]: Delete but i left it

File: pihole-google.txt (hunk 12 above)
Comment (gallegonovato) [reply pihole-google.txt:7663]: Done

File: pihole-google.txt (hunk 1 above)
Comment (gallegonovato) [reply pihole-google.txt]: The changes should be made

**Comment by gallegonovato** [comment]
The changes should be made now ;)

**Comment by gallegonovato** [comment]
Good morning:

Changes made. :)

I don't know what you think about making a GAFAM , of a hosts to avoid the Windows stuff like https://github.com/nickspaargaren/no-amazon/issues/4 comments.

Best regards

**Comment by gallegonovato** [comment]
good afternoon: I don't understand why it puts duplicates. I don't see any. And in this https://github.com/nickspaargaren/no-google/pull/115/commits/5eabcac7a01ce2384aaf04fcae63e51d635feb5b he put it all right. I looked several times and I can't find anything duplicate. In this https://github.com/nickspaargaren/no-google/pull/115/commits/0a15944eba5d5d35a75b04f1d90090c4485329d6 I don't know about lowercase? A hug

File: pihole-google.txt [hunk 22]
```diff
@@ -830,6 +830,451 @@ www.google.tt
...
+securepubads.g.doubleclick.net 
+ad.doubleclick.net 
+clients1.google.com
+adservice.google.de
+adservice.google.com
+adservice.google.es 
+ade.googlesyndication.com 
+id.google.de
+id.google.es
+e.crashlytics.com
```
Comment (nickspaargaren) [inline pihole-google.txt:1231]:
```suggestion
```

Found duplicate on line [1213](https://github.com/nickspaargaren/no-google/pull/115/files#diff-7bbfb1deafb933e9cdd519efb2376c63f2bec85a0e17a6a9a18138915e971150R1213).

File: pihole-google.txt [hunk 23]
```diff
@@ -830,6 +830,451 @@ www.google.tt
...
+adservice.google.de
+adservice.google.com
+adservice.google.es 
+ade.googlesyndication.com 
+id.google.de
+id.google.es
+e.crashlytics.com 
+reports.crashlytics.com 
+https://www.gmail.com
+https://gmail.com
```
Comment (nickspaargaren) [inline pihole-google.txt:1234]:
```suggestion
```
Domains including http:// and https:// are not valid

File: pihole-google.txt [hunk 24]
```diff
@@ -830,6 +830,451 @@ www.google.tt
...
+gmail.com
+https://meet.google.com
+https://workspace.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:1237]:
```suggestion
```
Domains including http:// and https:// are not valid

File: pihole-google.txt [hunk 25]
```diff
@@ -830,6 +830,451 @@ www.google.tt
...
+play.google.com/store
```
Comment (nickspaargaren) [inline pihole-google.txt:1238]:
```suggestion
```
Domains including a path are not valid

Comment (gallegonovato) [reply pihole-google.txt]: Delete

File: pihole-google.txt [hunk 26]
```diff
@@ -830,6 +830,451 @@ www.google.tt
...
+adservice.google.es 
+ade.googlesyndication.com 
+id.google.de
+id.google.es
+e.crashlytics.com 
+reports.crashlytics.com 
+https://www.gmail.com
+https://gmail.com
+gmail.com
+https://meet.google.com
```
[the remark this answers is not in the document]
Comment (gallegonovato) [reply pihole-google.txt]: Delete

File: pihole-google.txt (hunk 24 above)
Comment (gallegonovato) [reply pihole-google.txt]: Delete

File: pihole-google.txt [hunk 27]
```diff
@@ -830,6 +830,451 @@ www.google.tt
...
+clients1.google.com
+adservice.google.de
+adservice.google.com
+adservice.google.es 
+ade.googlesyndication.com 
+id.google.de
+id.google.es
+e.crashlytics.com 
+reports.crashlytics.com 
+https://www.gmail.com
```
[the remark this answers is not in the document]
Comment (gallegonovato) [reply pihole-google.txt]: Delete

File: pihole-google.txt (hunk 22 above)
Comment (gallegonovato) [reply pihole-google.txt]: Delete

File: pihole-google.txt [hunk 28]
```diff
@@ -830,6 +829,444 @@ www.google.tt
...
+www6.google.com
+gfe.core.l.google.com
+www9.google.com
+yp.google.com 							 								
+settings.crashlytics.com 							 								reports.crashlytics.com
+https://news.google.com
+https://news.google.com/publications
+https://maps.google.com
+https://mymaps.google.com
+https://developers.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:1216]:
```suggestion
```

Comment (gallegonovato) [reply pihole-google.txt]: Delete

File: pihole-google.txt [hunk 29]
```diff
@@ -830,6 +829,443 @@ www.google.tt
...
+mtalk.google.com 							 								
+alt1-mtalk.google.com 							 								alt1.mobile-gtalk.l.google.com 							 								alt2-mtalk.google.com 							 								alt2.mobile-gtalk.l.google.com 							 								alt3-mtalk.google.com 							 								alt3.mobile-gtalk.l.google.com 							 								alt4-mtalk.google.com 							 								alt4.mobile-gtalk.l.google.com 							 								alt5-mtalk.google.com 							 								alt5.mobile-gtalk.l.google.com 							 								alt6-mtalk.google.com 							 								alt6.mobile-gtalk.l.google.com 							 								alt7-mtalk.google.com 							 								alt7.mobile-gtalk.l.google.com 							 								alt8-mtalk.google.com 							 								alt8.mobile-gtalk.l.google.com
+googleads.g.doubleclick.net 							 								static.doubleclick.net
+pagead*.googlesyndication.com
```
Comment (nickspaargaren) [inline pihole-google.txt:1230]:
```suggestion
```

File: pihole-google.txt [hunk 30]
```diff
@@ -830,6 +829,443 @@ www.google.tt
...
+ade.googlesyndication.com
+tpc.googlesyndication.com
+googleadservices.com
+imageads.googleadservices.com
+imageads*.googleadservices.com
```
Comment (nickspaargaren) [inline pihole-google.txt:1235]:
```suggestion
```

File: pihole-google.txt [hunk 31]
```diff
@@ -830,6 +829,443 @@ www.google.tt
...
+mtalk.google.com 							 								
+alt1-mtalk.google.com 							 								alt1.mobile-gtalk.l.google.com 							 								alt2-mtalk.google.com 							 								alt2.mobile-gtalk.l.google.com 							 								alt3-mtalk.google.com 							 								alt3.mobile-gtalk.l.google.com 							 								alt4-mtalk.google.com 							 								alt4.mobile-gtalk.l.google.com 							 								alt5-mtalk.google.com 							 								alt5.mobile-gtalk.l.google.com 							 								alt6-mtalk.google.com 							 								alt6.mobile-gtalk.l.google.com 							 								alt7-mtalk.google.com 							 								alt7.mobile-gtalk.l.google.com 							 								alt8-mtalk.google.com 							 								alt8.mobile-gtalk.l.google.com
+googleads.g.doubleclick.net 							 								static.doubleclick.net
```
[the remark this answers is not in the document]
Comment (gallegonovato) [reply pihole-google.txt]: Done

File: pihole-google.txt (hunk 30 above)
Comment (gallegonovato) [reply pihole-google.txt]: Done

**Comment by gallegonovato** [comment]
I change this for this: https://github.com/nickspaargaren/no-google/pull/116

File: pihole-google.txt [hunk 32]
```diff
@@ -830,6 +829,465 @@ www.google.tt
...
+www.news.google.com
+www10.google.com
+webdrive-client.l.google.com
+www4.google.com
+www5.google.com
+www5.l.google.com
+www6.google.com
+gfe.core.l.google.com
+www9.google.com
+yp.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:1210]:
```suggestion
yp.google.com
```

Comment (nickspaargaren) [reply pihole-google.txt:1210]: The problem is that there are a few domains with a lot spacings.

Comment (gallegonovato) [reply pihole-google.txt]: Done

File: pihole-google.txt [hunk 33]
```diff
@@ -830,26 +829,461 @@ www.google.tt
...
+www3.l.google.com
+www-wide.l.google.com
+search.google.com
+feeds.google.com
+uploads.google.com
+feedsftp.l.google.com
+www5f.google.com
+admin.google.com
+maven.google.com
+googlecn-lopri.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:881]: We discovered that `*.l.google.com` has a lot of subdomains, including random strings. It is impossible to put all these domains in the blocklist. That's why we created a few regexes in addition to the list, you can find them here:

https://github.com/nickspaargaren/no-google#important-regex-filters-not-included-in-blocklist

Please remove all `*.l.google.com` domains from the pull request and optionally add the regex to your blocking software solution.

Comment (gallegonovato) [reply pihole-google.txt]: deleted everything that included .l.google.com

**Comment by gallegonovato** [comment]
good afternoon @nickspaargaren 

Isn't it worth these suggestions that I have made? 

A hug

**Comment by nickspaargaren** [comment]
You are the best, merged in https://github.com/nickspaargaren/no-google/commit/325f2b9c1bdd76cf777f080ac332b585b327a334. I just removed a few that were not Google and did some sorting!

**Comment by gallegonovato** [comment]
No problem. I thought I had done it wrong and it was worthless. You are the genius, I am a mere apprentice. A hug

**Pull request merged**